torchvision/datasets/utils.py

```diff
@@ -483,3 +484,39 @@ def verify_str_arg(
         raise ValueError(msg)
 
     return value
+
+
+def read_pfm(file_name: str, slice_channels: int = 2) -> np.ndarray:
```

Comment:
Let's keep it private, we don't really need to expose it  to users

```suggestion
def _read_pfm(file_name: str, slice_channels: int = 2) -> np.ndarray:
```